clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -96,55 +203,197 @@
+  },
+  "CHARACTERS": {
+    "prefix": "CHARACTERS",
+    "body": [
+      "ENVIRONMENT DIVISION.",
+      "CONFIGURATION SECTION.",
+      "SYMBOLIC ${1|CHARACTER,CHARACTERS|} ${2:symbolic-character}"
+    ],
+    "description": "SYMBOLIC CHARACTER"
+  },
+  "CLASS DIGIT IS": {
+    "prefix": "CLASS",
+    "body": ["CLASS DIGIT IS \"0\" THROUGH \"9\""],
+    "description": ""
   },
   "CLOSE": {
     "prefix": "CLOSE",
     "body": ["CLOSE $(1:fileName}"],
     "description": "CLOSE filename"
   },
+  "CODE SET": {
+    "prefix": "CODE-SET",
+    "body": [
+      "DATA DIVISION.",
+      "FILE SECTION.",
+      "FD CODE-SET IS ${1:ALPHABET-VAR-NM}.",
+      "$0"
+    ],
+    "description": "CODE-SET is used in the COBOL DATA DIVISION, FILE SECTION, FD CLAUSE. https://www.ibm.com/docs/en/SS6SG3_6.4.0/pdf/pgmvs.pdf"
+  },
+  "COLLATING": {
+    "prefix": "COLLATING",
+    "body": ["PROGRAM COLLATING SEQUENCE IS ${1|EBDIC,ASCII|}"],
+    "description": "Part of the IDENTIFICATION DIVISION. Note that the default option is EBCDIC"
+  },
+  "COMP": {
+    "prefix": "COMP",
+    "body": ["77 ${1:VAR-BIN} PIC S9(4) ${2|COMP,COMPUTATIONAL|}."],
+    "description": "BINARY data and synonyms COMP and COMP-4 are the two's complement data representation in COBOL. Two's complement Hex data (EBCDIC) representation of numeric values in fields within the DATA DIVISION."
+  },
+  "COMP-1": {
+    "prefix": "COMP-1",
+    "body": [
+      "77 ${1:VAR-FLOAT-SINGLE-PREC} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-1."
+    ],
+    "description": "A four-byte single-precision floating-point number declared in the DATA DIVISION. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=arithmetic-formats-numeric-data"
+  },
+  "COMP-2": {
+    "prefix": "COMP-2",
+    "body": [
+      "77 ${1:VAR-FLOAT-DOUBLE-PREC} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-2."
+    ],
+    "description": "A four-byte double-precision floating-point number declared in the DATA DIVISION. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=arithmetic-formats-numeric-data"
+  },
+  "COMP-3": {
+    "prefix": "COMP-3",
+    "body": [
+      "77 ${1:VAR-PACKED-NUM-DATA} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-3."
+    ],
+    "description": "Internally-packed (efficient) number declared in the DATA DIVISION. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=6-packed-decimal-comp-3"
+  },
+  "COMP-4": {
+    "prefix": "COMP-4",
+    "body": ["77 ${1:VAR-BIN} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-4."],
+    "description": "BINARY data and synonyms COMP and COMP-4 are the two's complement data representation in COBOL. Two's complement Hex data (EBCDIC) representation of numeric values in fields within the DATA DIVISION."
```

Comment:
"BINARY, and its synonyms COMP and COMP-4, are used next to numeric fields in the DATA DIVISION to specify that the data is represented in two's complement hex (EBCDIC) format."